assets/js/utils/channelUtils.ts

```diff
@@ -102,3 +103,11 @@ export const syncPresenters = (
     eventChannel.push("sync_presence", {});
   }
 };
+
+export const getChannelId = (): string => window.location.pathname.split("/")[2];
+
+const compareParticipants = (x: Participant, y: Participant): number => {
+  if (x.isModerator != y.isModerator) return x.isModerator ? -1 : 1;
+  if (x.isPresenter != y.isPresenter) return x.isPresenter ? -1 : 1;
+  return 0;
+};
```

Comment:
I think it is too complex I would create a function e.g: `participantToNumber` which takes the participant and return rank of this participant. Then this function could look like this:
```suggestion
const compareParticipants = (x: Participant, y: Participant): number => {
return participantToNumber(x) - participantToNumber(y)
};
```